### PR TITLE
vimpc: new package at 0.09.2

### PR DESCRIPTION
**Pull request by cemkeylan**
Read: https://k1ss.org/style.txt
Read: https://k1ss.org/guidestones.txt

---

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/vimpc/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./autogen.sh
```
Comment (dylanaraps) [inline community/vimpc/build]: https://k1ss.org/style [0401]

File: community/vimpc/depends [hunk 2]
```diff
@@ -0,0 +1,4 @@
+automake make
+libmpdclient
+ncurses make
```
Comment (dylanaraps) [inline community/vimpc/depends]: Runtime dep?

Comment (cemkeylan) [reply community/vimpc/depends]: Yeah libmpdclient is a runtime dependency for all mpd clients.

File: community/vimpc/build (hunk 1 above)
Comment (cemkeylan) [reply community/vimpc/build]: There aren't any sources for this.

File: community/vimpc/depends (hunk 2 above)
Comment (dylanaraps) [reply community/vimpc/depends]: I meant `ncurses` being `make`.

File: community/vimpc/build (hunk 1 above)
Comment (dylanaraps) [reply community/vimpc/build]:
```
|  If autogen.sh or autoreconf are required, prefer     |
|  autoreconf.                                          |
|               
```

**Pull request merged**